src/cryptoadvance/specter/templates/includes/address-label.html

```diff
@@ -1,73 +1,104 @@
 <template id="address-label">
     <style>
         .address-label-form {
-            display: inline;
+            display: flex;
         }
+
         .service-icon {
-            margin-top: -5px;
-            height:24px;
+            margin-left: 2px;
+            margin-right: 2px;
+            height: 24px;
             vertical-align: middle;
         }
+
         .label {
             word-break: break-all;
             background: none;
             color: #fff;
             font-size: 1em;
             max-width: 80%;
             outline: none;
+            margin-top: 1px;
+            margin-left: 2px;
+            white-space: nowrap;
```

Comment:
This causes that the addresses here are not fully displayed:

<img width="495" alt="grafik" src="https://user-images.githubusercontent.com/70536101/203608195-122c1e53-8bf4-4ffd-8745-a6cf59920896.png">

We could keep the `nowrap` and just shorten the addresses like we do in the addresses overview:

<img width="380" alt="grafik" src="https://user-images.githubusercontent.com/70536101/203608378-1918ae22-9321-4114-b7a9-6f217a925489.png">